mesonbuild/backend/ninjabackend.py

```diff
@@ -1178,6 +1172,14 @@ def generate_rust_target(self, target, outfile):
         if isinstance(target, build.SharedLibrary):
             self.generate_shsym(outfile, target)
 
+    @staticmethod
+    def get_crstr(for_machine: MachineChoice):
```

Comment:
I only internally use autoconf-style suffixes in the ninja file. Normal env vars unaffected.